src/System.Data.OleDb/src/Configurations.props

```diff
@@ -7,7 +7,7 @@
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations);
-      netfx-Windows_NT;
+      _netfx-Windows_NT;
```

Comment:
Why are you removing the vertical build for netfx but keeping the package build for net461?  I would expect them both to be gone (and harvesting) or neither.